src/main/java/uk/gov/hmcts/reform/prl/controllers/DraftAnOrderController.java

```diff
@@ -66,9 +74,22 @@ public AboutToStartOrSubmitCallbackResponse populateHeader(
             callbackRequest.getCaseDetails().getData(),
             CaseData.class
         );
+
         Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
         caseDataUpdated.put("selectedOrder", caseData.getCreateSelectOrderOptions() != null
             ? caseData.getCreateSelectOrderOptions().getDisplayedValue() : "");
+
```

Comment:
Can we please move any business logic inside the service class

Reply:
will do it later. for now will raise a ticket and put in backlog.